v0.42 is changing pane arrangement strategies on mouse movement

## Issue description

Zellij v0.42.0 is changing arrangement strategies on mouse movement.
I have a single line config, `pane_frames false`.

## Minimal reproduction

minimal setup to reproduce it: open a second pane and move the mouse over your terminal.

## Other relevant information

I don't know if it's relevant, but that's on Windows 11 WSL with Ubuntu using Microsoft Terminal.

`termwiz` builds just fine but for some reason the `zellij` build complains...
```
💾 🏡/dev/wezterm/termwiz  HEAD (8b7ca45) +1 at 2025-03-19 20:19:14
❯ git log -1
commit 8b7ca454b5e0a65b69b1b736db54eae3c195384b (HEAD, tag: termwiz-0.23.2, tag: termiz-0.23.2)
Author: Wez Furlong <wez@wezfurlong.org>
Date:   Wed Mar 19 07:23:49 2025 -0700

    termwiz: bump version

💾 🏡/dev/wezterm/termwiz  HEAD (8b7ca45) +1 at 2025-03-19 20:19:15
❯ cargo build
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.14s
```
```
💾 🏡/dev/zellij  main +2 ?2 at 2025-03-19 20:16:25
❯ cargo build -j1
   Compiling termwiz v0.23.2
error: proc-macro derive panicked
  --> /home/zeev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termwiz-0.23.2/src/tmux_cc/mod.rs:12:14
   |
12 |     #[derive(Parser)]
   |              ^^^^^^
   |
   = help: message: grammar error

             --> 24:1
              |
           24 | continue = { "%continue " ~ pane_id }␊
              | ^------^
              |
              = continue is a rust keyword

error[E0432]: unresolved import `parser::Rule`
 --> /home/zeev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termwiz-0.23.2/src/tmux_cc/mod.rs:2:5
  |
2 | use parser::Rule;
  |     ^^^^^^^^^^^^ no `Rule` in `tmux_cc::parser`
  |
help: consider importing this struct instead
  |
2 | use crate::hyperlink::Rule;
  |     ~~~~~~~~~~~~~~~~~~~~~~

error[E0599]: no function or associated item named `parse` found for struct `TmuxParser` in the current scope
   --> /home/zeev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termwiz-0.23.2/src/tmux_cc/mod.rs:219:41
    |
14  |     pub struct TmuxParser;
    |     --------------------- function or associated item `parse` not found for this struct
...
219 |     let mut pairs = parser::TmuxParser::parse(Rule::line_entire, line)?;
    |                                         ^^^^^ function or associated item not found in `TmuxParser`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `parse`, perhaps you need to implement one of them:
            candidate #1: `nom::internal::Parser`
            candidate #2: `nom::sequence::Tuple`
            candidate #3: `pest::Parser`

error[E0599]: no function or associated item named `parse` found for struct `TmuxParser` in the current scope
   --> /home/zeev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termwiz-0.23.2/src/tmux_cc/mod.rs:786:37
    |
14  |     pub struct TmuxParser;
    |     --------------------- function or associated item `parse` not found for this struct
...
786 |     let pairs = parser::TmuxParser::parse(Rule::layout_window, layout)?;
    |                                     ^^^^^ function or associated item not found in `TmuxParser`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `parse`, perhaps you need to implement one of them:
            candidate #1: `nom::internal::Parser`
            candidate #2: `nom::sequence::Tuple`
            candidate #3: `pest::Parser`

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `termwiz` (lib) due to 4 previous errors
```

--- a/zellij-utils/Cargo.toml
+++ b/zellij-utils/Cargo.toml
@@ -47,7 +47,7 @@ vte = { version = "0.11.0", default-features = false }
 
 #[cfg(not(target_family = "wasm"))]
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
-termwiz = "0.22"
+termwiz = "0.23.2"
 log4rs = "1.2.0"
 signal-hook = "0.3"
 interprocess = "1.2.1"

